src/models/confirmRules.model.js

```diff
@@ -6,10 +6,32 @@ const BaseModel = require('./base.model')
 const LoggingService = require('../services/logging.service')
 const ApplicationLine = require('./applicationLine.model')
 
-module.exports = class Site extends BaseModel {
+module.exports = class ConfirmRules extends BaseModel {
   constructor (data) {
     super()
-    this.applicationLineId = data.applicationLineId
+    // Merge the data with this instance
+    Object.assign(this, data)
```

Comment:
As discussed I think we should keep using the non-ES6 syntax to set the instance properties in order to aid less experienced Javascript developers and to make it clear what the properties of the class are.

i.e.
The existing method of doing:
this.something = obj.something